From 5255e844afb7b2618cfdbe9a724d95382dc203f8 Mon Sep 17 00:00:00 2001
From: Jeremy Huddleston <jeremyhu@apple.com>
Date: Mon, 13 Jun 2011 12:00:55 -0700
Subject: [PATCH] apple: applegl_destroy_context: Pass along the correct
 display

Signed-off-by: Jeremy Huddleston <jeremyhu@apple.com>
(cherry picked from commit c6cf82fb553405bee241170f6e51cafd751d46ea)
---
 src/glx/applegl_glx.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/glx/applegl_glx.c b/src/glx/applegl_glx.c
index 9b8605f..e888a0f 100644
--- a/src/glx/applegl_glx.c
+++ b/src/glx/applegl_glx.c
@@ -43,7 +43,7 @@
 static void
 applegl_destroy_context(struct glx_context *gc)
 {
-   apple_glx_destroy_context(&gc->driContext, gc->currentDpy);
+   apple_glx_destroy_context(&gc->driContext, gc->psc->dpy);
 }
 
 static int
-- 
1.7.5.4

